source/blog/2019-05-21-monetizing-open-source.md

```diff
@@ -0,0 +1,72 @@
+of our monetization efforts.
+
+## Patreon
+
+Patreon is a platform that allows open-source maintainers, artists, creators, etc. to create
+relationships with their users and offer them benefits for becoming a Patreon.
+
+If you would like to make a donation to the project, Patreon is the place to do it.
+Depending on the tier, we offer a few different benefits:
+
+- A mention on Twitter.
+- Your name and website link on the Doctrine website.
+- Priority responses to Stackoverflow or GitHub Issues/Pull Requests.
+- Placement on a Partners page on the Doctrine website, with your logo, link and a description advertising your company's services.
+
+[Become a Patreon](https://www.patreon.com/phpdoctrine) of Doctrine today!
+
+## Consulting
+
+Under the Doctrine Company, we are providing [Consulting and Training](/consulting.html)
+services to companies that use PHP. The services we offer are not limited to Doctrine itself.
+Our team specializes in writing high quality PHP. Whether you are starting a new project or
+modernizing a legacy application, we can help your team level up their skills with tools
+like the following:
+
+- [Doctrine DBAL](/projects/dbal.html) - Connect to your favorite RDBMS using the Doctrine DBAL.
+- [Doctrine Database Migrations](/projects/migrations.html) - Manage and keep your database schema up to date safely with Doctrine Migrations.
+- [Doctrine ORM](/projects/orm.html) - Represent your domain model cleanly with plain old PHP objects and ensure the integrity of your data.
+- [Doctrine Coding Standard](/projects/coding-standard.html) - Never again debate about code aesthetics in pull request reviews. Implement the Doctrine Coding Standard and the rules will be enforced in your build process.
+- [Psalm](https://psalm.dev/) / [PHPStan](https://github.com/phpstan/phpstan) - Statically analyze your codebase and find bugs that would otherwise only be found at runtime in production.
+
+If you are interested, take a look at our [Consulting](/consulting.html) page and contact
+us at [consulting@doctrine-project.org](mailto:consulting@doctrine-project.org) for more
+information.
+
+## Events
+
+We will host regular monthly webinars using Zoom. Each month we will have different topics
+presented to you by Doctrine Core team members and members of the community. We already have
+some great topics lined up for the next few months:
+
+- [Doctrine for Beginners on June 25th 2019](/events/2/doctrine-for-beginners.html)
+- [Getting started with Doctrine MongoDB ODM on July 16th 2019](/events/3/getting-started-with-doctrine-mongodb-odm.html)
+- [PHP Internals for the Inquisitive Developer on September 10th 2019](/events/1/php-internals-for-the-inquisitive-developer.html)
+
+## Carbon Ads
+
+One of the most valuable assets we have besides software is our website and the traffic we
+receive every month. You may have noticed some subtle ad placements on our website. These ads
```

Comment:
Would it help to include something that mentions the ads are text-based"? Perhaps: "These text-based ads are provided..."